### ISSUE TITLE
libsteam_api.so replacement

### ISSUE THREAD

**Issue by scarburato** [SwagSoftware/Kisak-Strike#35]
So I tried adding compile definition `NO_STEAM` to compile without libsteam but it gave a lot of errors during compile and I gave up.

Then I tried to replace `libsteam_api.so` with [Goldberg's implementation](https://gitlab.com/Mr_Goldberg/goldberg_emulator) and it worked. And I was able to run the game without the game exiting on `[S_API FAIL] SteamAPI_Init() failed; create pipe failed.FATAL ERROR: Failed to connect with local Steam Client process!` like with the repo's bundled library.

So, would it be ok to switch to the Steam-free implementation for the time being?

**Comment by LWSS** [comment]
yeah I think that's what the russians do for their elbrus project, been meaning to try that out actually

**Comment by r-a-sattarov** [comment]
> yeah I think that's what the russians do for their elbrus project

Yeap, I use it to compile and run on Elbrus PC.
